client/src/helpers/APICalls/convertDateRightFormat.ts

```diff
@@ -0,0 +1,21 @@
+export const convertDate = (date: string): string => {
+  const monthsInString = [
+    ,
+    'January',
+    'February',
+    'Marth',
+    'April',
+    'May',
+    'June',
+    'July',
+    'August',
+    'September',
+    'October',
+    'November',
+    'December',
+  ];
+
+  const dateArray = date.split('/');
+
+  return `${dateArray[1]} ${monthsInString[Number(dateArray[0])]} ${dateArray[2]}`;
```

Comment:
Could you elaborate on this here more?

Reply:
Am Trying to created a desired date format on the template